package.json

```diff
@@ -1,61 +1,83 @@
 {
-  "name": "@metamask/module-template",
-  "version": "0.0.0",
-  "description": "The MetaMask Node module template.",
+  "name": "@metamask/openrpc-inspector-transport",
+  "version": "0.0.0-development",
+  "homepage": "https://metamask.github.io/iframe-ee-openrpc-inspector-transport",
+  "description": "",
+  "main": "build/index.js",
   "repository": {
     "type": "git",
-    "url": "https://github.com/MetaMask/metamask-module-template.git"
+    "url": "https://github.com/MetaMask/iframe-ee-openrpc-inspector-transport.git"
   },
-  "main": "dist/index.js",
-  "types": "dist/index.d.ts",
-  "files": [
-    "dist/"
-  ],
   "scripts": {
     "setup": "yarn install && yarn allow-scripts",
-    "test": "jest",
-    "test:watch": "jest --watch",
-    "prepublishOnly": "yarn build:clean && yarn lint && yarn test",
+    "start": "yarn typings && webpack serve --progress --mode='development'",
+    "build": "yarn typings && webpack --mode production",
+    "typings": "open-rpc-typings -d src/openrpc.json --output-ts src/__GENERATED_TYPES__/",
     "lint:eslint": "eslint . --cache --ext js,ts",
     "lint:misc": "prettier '**/*.json' '**/*.md' '!CHANGELOG.md' '**/*.yml' --ignore-path .gitignore",
     "lint": "yarn lint:eslint && yarn lint:misc --check",
     "lint:fix": "yarn lint:eslint --fix && yarn lint:misc --write",
-    "build:clean": "rimraf dist && yarn build",
-    "build": "tsc --project ."
+    "test": "yarn typings && yarn lint"
   },
+  "author": "",
   "devDependencies": {
     "@lavamoat/allow-scripts": "^1.0.5",
     "@metamask/auto-changelog": "^2.3.0",
     "@metamask/eslint-config": "^7.0.1",
     "@metamask/eslint-config-jest": "^7.0.0",
     "@metamask/eslint-config-nodejs": "^7.0.1",
     "@metamask/eslint-config-typescript": "^7.0.1",
-    "@types/jest": "^26.0.13",
-    "@typescript-eslint/eslint-plugin": "^4.21.0",
-    "@typescript-eslint/parser": "^4.21.0",
-    "eslint": "^7.23.0",
-    "eslint-config-prettier": "^8.1.0",
-    "eslint-plugin-import": "^2.22.1",
-    "eslint-plugin-jest": "^24.3.4",
+    "@open-rpc/typings": "^1.9.0",
+    "@types/jest": "^24.0.13",
+    "@types/pump": "^1.1.1",
+    "@types/react-dom": "^16.8.4",
+    "@typescript-eslint/eslint-plugin": "^4.28.3",
+    "@typescript-eslint/parser": "^4.28.3",
+    "eslint": "^7.30.0",
+    "eslint-config-prettier": "^8.3.0",
+    "eslint-plugin-import": "^2.23.4",
+    "eslint-plugin-jest": "^24.3.6",
     "eslint-plugin-node": "^11.1.0",
-    "eslint-plugin-prettier": "^3.3.1",
-    "jest": "^26.4.2",
+    "eslint-plugin-prettier": "^3.4.0",
+    "html-webpack-plugin": "^5.3.2",
+    "jest": "^24.9.0",
+    "node-polyfill-webpack-plugin": "^1.1.4",
     "prettier": "^2.2.1",
-    "prettier-plugin-packagejson": "^2.2.11",
```

Comment:
Well, that explains it! I was confused about why there were no `dependencies`. They're just sorted weird. The `devDependencies`-first ordering is the only one that ever throws me.

Reply:
i added it back and ran lint:fix, didnt seem to change the ordering of devdeps vs deps